### PR TITLE
TW-1795: Change style loading dialog

### PR DESCRIPTION
**Pull request by nqhhdev**
### Ticket:

- #1795 

### Need merge:

- https://github.com/linagora/future-loading-dialog/pull/1

### Resolved

https://github.com/linagora/twake-on-matrix/assets/99852347/1e20e479-6a83-4378-a1cf-23b21c316cdd

https://github.com/linagora/twake-on-matrix/assets/99852347/4e9b3c34-fcce-4cbf-8dcc-2715b8cc7465

### CODE REVIEW

File: lib/utils/dialog/twake_dialog.dart [hunk 1]
```diff
@@ -118,22 +160,32 @@ class TwakeDialog {
...
-      content: Row(
+      backgroundColor: Colors.transparent,
+      content: Column(
+        mainAxisSize: MainAxisSize.min,
         children: [
-          const Padding(
-            padding: EdgeInsets.only(right: 16.0),
-            child: CircularProgressIndicator.adaptive(),
+          LottieBuilder.asset(
+            'assets/twake_loading.json',
```
Comment (hieutbui) [inline lib/utils/dialog/twake_dialog.dart:178]: How about store it in a variable?

Comment (nqhhdev) [reply lib/utils/dialog/twake_dialog.dart]: Done

File: lib/resource/image_paths.dart [hunk 2]
```diff
@@ -57,4 +58,8 @@ class ImagePaths {
   static String getConfigurationImagePath(String imageName) {
     return AssetsPaths.configurationImages + imageName;
   }
+
+  static String _getAssetPath(String imageName) {
+    return AssetsPaths.assets + imageName;
+  }
```
Comment (sherlockvn) [inline lib/resource/image_paths.dart:64]:
```suggestion
  static String _getAssetPath(String assetName) {
    return AssetsPaths.assets + assetName;
  }
```

File: lib/utils/dialog/twake_dialog.dart [hunk 3]
```diff
@@ -54,6 +65,38 @@ class TwakeDialog {
     return await showFutureLoadingDialog(
       context: twakeContext,
       future: future,
+      loadingIcon: LottieBuilder.asset(
```
Comment (sherlockvn) [inline lib/utils/dialog/twake_dialog.dart:68]: IMO, if it have two many condition, create a widget for web and mobile

**Comment by sherlockvn** [comment]
Test:
- [ ] web
- [ ] ios

**Review by sherlockvn — APPROVED** [review]

**Comment by hieutbui** [comment]
Tested:
- [X] Android

**Review by hieutbui — APPROVED** [review]

**Comment by nqhhdev** [comment]
Rebased, ready to merge

**Comment by Te-Z** [comment]
The overlay is black but should be white, and I don't think the opacity is 75% as expected -> https://www.figma.com/design/uLw3iWnOEGaRlJnfPz4Pv0/Twake-%5BMatrix%5D?node-id=24467-26025&t=Vp0Y5QOhe8YHr3hI-4

**Comment by nqhhdev** [comment]
> The overlay is black but should be white, and I don't think the opacity is 75% as expected -> https://www.figma.com/design/uLw3iWnOEGaRlJnfPz4Pv0/Twake-%5BMatrix%5D?node-id=24467-26025&t=Vp0Y5QOhe8YHr3hI-4

Updated, please review and need merge first

https://github.com/linagora/future-loading-dialog/pull/1

**Review by sherlockvn — APPROVED** [review]

**Review by hieutbui — APPROVED** [review]

**Comment by Te-Z** [comment]
> > The overlay is black but should be white, and I don't think the opacity is 75% as expected -> https://www.figma.com/design/uLw3iWnOEGaRlJnfPz4Pv0/Twake-%5BMatrix%5D?node-id=24467-26025&t=Vp0Y5QOhe8YHr3hI-4
> 
> Updated, please review and need merge first
> 
> [linagora/future-loading-dialog#1](https://github.com/linagora/future-loading-dialog/pull/1)

done

**Review by Te-Z — APPROVED** [review]

**Comment by Te-Z** [comment]
waiting for the update of pubspec.yaml to merge

**Comment by nqhhdev** [comment]
> waiting for the update of pubspec.yaml to merge

Done, approve again and merge

**Review by hieutbui — APPROVED** [review]

**Pull request merged**